cfp.md

```diff
@@ -1,9 +1,64 @@
 ---
 layout: page
-title: "BSidesSF 2019 Call For Papers"
+title: "BSidesSF 2020 Call For Participation"
 permalink: /cfp.html
+redirect_from: "/cfw"
+redirect_from: "/cfw.html"
 --- 
 
-The BSidesSF 2019 CFP is now closed. See all our amazing speakers March 3-4, 2019 at the City View at Metreon in San Francisco. 
+**What:** BSidesSF 2020 Call For Participation for Talks and Workshops
 
-As always -- Questions? E-mail cfp [at] bsidessf.org
+BSidesSF is a non-profit organization designed to advance the body of Information Security knowledge, by providing an annual, open forum for discussion and debate for security practitioners. We produce a conference that is a source of education, collaboration, and continued conversation for information technologists. The technical and academic presentations at BSidesSF are given in the spirit of peer review and advanced knowledge dissemination. This allows the field of Information Security to grow in breadth and depth, and continue in its pursuit of highly advanced, scientifically based knowledge. 
+
+We are a practitioner focused event that attracts a wide range of attendees ranging from engineers & security leaders to executive decision makers. This is a great opportunity for you to showcase your work and share your ideas with industry peers. Bonus points if you can tie your presentation to this year’s conference theme: San Francisco! 
+
+The BSides community has continuously raised the bar and put the info back in infosec. We thank each and every member/participant/organizer of this community for their hard work, sweat, and relentless pursuit of making high quality content accessible. We celebrate diverse approaches and points of view and believe that our commitment to diversity fosters innovation. BSidesSF is continuing to build upon a culture that drives our community to include a wide range of thoughts and ideas.
+
+**When:**  BSidesSF will take place in San Francisco from February 22-24, 2020.  The Call for Participation for talks and workshops will close on November 6, 2019 at 11:59 PM PST.
```

Comment:
For*

Reply:
I'm missing something here. Are you saying the "for" in `Call for Participation` should be capitolized to `Call For Participation`? Something else?

Reply:
Exactly. :-)

Reply:
Ah, I see the pattern requested elsewhere. Updated to `Call For Participation`